pkg/network/etw/etw-http-service.go

```diff
@@ -746,8 +751,9 @@ func httpCallbackOnHTTPRequestTraceTaskDeliver(eventInfo *C.DD_ETW_EVENT_INFO) {
 		return
 	}
 
-	httpConnLink.http.AppPool = appPool
+	httpConnLink.http.AppPool = strings.Replace(appPool, " ", "_", -1)
```

Comment:
why the change in how the `appPool` is stored?

Reply:
because it becomes a tag and it can't have spaces.

Reply:
can you rename the `http.AppPool` and `http.SiteName` attributes as well, to indicate they contain tag names and not the actual application pool and site name.

Reply:
I actually do not like renaming functions or vars things here (in this module). They are converted to tags at different layer. Perhaps that where it should be replace spaces with underscores.